adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -117,6 +119,7 @@ public void export(CannotDownloadArchiveFromNetworkNotice toExport) throws IOExc
         protoBuilder.setCsvFileName(toExport.getFilename())
                 .setType(GtfsValidationOutputProto.GtfsProblem.Type.TYPE_CSV_FILE_CORRUPTED)
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR)
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder